tests/models/test_lora_layers.py

```diff
@@ -212,3 +213,75 @@ def test_lora_save_load_legacy(self):
 
         # Outputs shouldn't match.
         self.assertFalse(torch.allclose(torch.from_numpy(orig_image_slice), torch.from_numpy(lora_image_slice)))
+
+    # copied from: https://colab.research.google.com/gist/sayakpaul/df2ef6e1ae6d8c10a49d859883b10860/scratchpad.ipynb
+    def get_dummy_tokens(self):
+        max_seq_length = 77
+
+        inputs = torch.randint(2, 56, size=(1, max_seq_length), generator=torch.manual_seed(0))
+
+        prepared_inputs = {}
+        prepared_inputs["input_ids"] = inputs
+        return prepared_inputs
+
+    def get_text_lora_attn_procs(self, text_encoder: nn.Module, randn_weight=False):
+        text_lora_attn_procs = {}
+        for name, module in text_encoder.named_modules():
+            if any(x in name for x in TEXT_ENCODER_TARGET_MODULES):
+                attn_proc = LoRAAttnProcessor(hidden_size=module.out_features, cross_attention_dim=None)
+                # set up.weights
+                for layer_name, layer_module in attn_proc.named_modules():
+                    if layer_name.endswith("_lora"):
+                        weight = (
+                            torch.randn_like(layer_module.up.weight)
+                            if randn_weight
+                            else torch.zeros_like(layer_module.up.weight)
+                        )
+                        layer_module.up.weight = torch.nn.Parameter(weight)
+                text_lora_attn_procs[name] = attn_proc
+        return text_lora_attn_procs
+
+    def test_text_encoder_lora_monkey_patch(self):
+        pipe = StableDiffusionPipeline.from_pretrained("runwayml/stable-diffusion-v1-5")
```

Comment:
Could we maybe use a smaller pipeline like the following?

https://github.com/huggingface/diffusers/blob/85eff637aad1106f593d7535ec41cdb736b0b2ea/tests/models/test_lora_layers.py#L140

Helps us to run the tests faster but does the job of proper testing at the same time.

Reply:
Fixed [1da772b](https://github.com/huggingface/diffusers/pull/3490/commits/1da772b9fe8f64702989c1319348dfffd65dc491)